amd/src/mod_hsuforum_loader.js

```diff
@@ -47,7 +47,8 @@ define(['jquery'], function ($) {
             $(posts).each(function(){
                 const postObserver = new MutationObserver(() => {
                     let form = $(this).find('form');
-
+                    let postid = $(this).closest(".hsuforum-post-wrapper").data('postid');
+                    $(this).closest(".hsuforum-post-wrapper").find('input[name="reply"]').val(postid);
```

Comment:
Have you tested this on both the forum view page and the discussion page?

Reply:
yep, tested on adding new topic and threaded replies